lib/android-helpers.js

```diff
@@ -890,5 +890,61 @@ helpers.isEmulator = function isEmulator (adb, opts) {
 helpers.bootstrap = Bootstrap;
 helpers.unlocker = unlocker;
 
-export { helpers, SETTINGS_HELPER_PKG_ID, prepareAvdArgs, ensureNetworkSpeed };
+const SERVER_ARGS = {
```

Comment:
this code basically repeats capability validators from the base driver. I don't see a need in such duplication

Reply:
See https://github.com/appium/appium-base-driver/blob/de7e41bbf7e7dc9c61749f094b997f95e73eef9b/lib/basedriver/capabilities.js#L24

Reply:
i think it's good to define the possible server args separately from the capabilities. but i agree, we could probably just use the existing caps validation logic on these. in addition, it's not necessary to put strings around key names in JS unless they contain a special character. pretty sure our lint will catch this if you run it.